docs/technical-design/launch-darkly-testing-approach.md

```diff
@@ -91,8 +217,6 @@ export REACT_APP_LD_CLIENT_ID='Place Launch Darkly ID here'
     })
   ```
 
-### Adding feature flags
-Adding a new feature flag for testing is automatically done once you add the flag into list of flags in the `flags.ts` constants file located in `app-web/src/common-code/featureFlags`.
+## Adding and removing feature flags
+Adding a new feature flag for testing is automatically done once you add the flag into list of flags in the `flags.ts` constants file located in `app-web/src/common-code/featureFlags`. Removing the flag from `flags.ts` can be done by removing it from the list. **For testing purposes we should not be removing or adding feature flags specifically for testing.**
```

Comment:
```suggestion
Adding a new feature flag for testing is automatically done once you add the flag into the list of flags in the `flags.ts` constants file located in `app-web/src/common-code/featureFlags`. Removing the flag from `flags.ts` can be done by removing it from the list. **We should not be removing or adding feature flags specifically for testing.**
```

Reply:
Can you say more about the bold-ed text - why not?

Reply:
I think I could have worded the bold text better. 

My thought was that this section of the text is just to give more context on how/where feature flags are added and removed. The reason for the bold text is to clarify the feature flags in `flags.ts` should not be edited specifically for the tests. Mainly removing flags or changing default values. Because all the LaunchDarkly testing implementations rely on those flags and default value for configuration. A change would cause many tests fail.

It's not super important because then you would have failing tests to let you know not to change those defaultValues.